firebase-perf/src/main/java/com/google/firebase/perf/gauges/CpuGaugeCollector.java

```diff
@@ -96,6 +97,7 @@ private CpuGaugeCollector() {
   }
 
   @VisibleForTesting
+  @Inject
   CpuGaugeCollector(
       ScheduledExecutorService cpuMetricCollectorExecutor,
       String fakeProcFileName,
```

Comment:
Can you update the name here? This is no longer a "fakeProcFileName".

Reply:
Done.